wormhole-connect/src/views/Bridge/Modals/TokensModal.tsx

```diff
@@ -363,13 +362,6 @@ function TokensModal() {
           </Collapse> */}
         </div>
       </Scroll>
-
-      <div className={classes.register}>
-        <div className={classes.registerText}>Don't see your token?</div>
-        <a href="#" className={classes.registerLink}>
-          Register token
-        </a>
-      </div>
```

Comment:
Really shouldn't prompt user to register tokens here, kind of a dumb ux. Can just show the "no results" for tokens with balances